<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCollection_VectorSelector_WidgetClass</class>
 <widget class="QWidget" name="CreateCollection_VectorSelector_WidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateCollection_VectorSelector_Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Plasmid</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="description">
     <property name="text">
      <string>Select or drag and drop the vector plasmid file. (File should be fasta format - *.fasta)</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLineEdit" name="plasmid_location"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="main_title">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Select the Vector Plasmid&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="window_title">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Create the New Collection&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QComboBox" name="plasmid_selector"/>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="location_select_btn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLineEdit" name="plasmid_name"/>
   </item>
   <item row="14" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="next_btn">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
